ngraph/core/include/ngraph/op/convert_like.hpp

```diff
@@ -43,6 +43,9 @@ namespace ngraph
                 virtual std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
             };
-        }
-    }
-}
+
+        } // namespace v1
+        using v1::ConvertLike;
```

Comment:
Please remove this `using`. We have such `using`s only for some v0 operations and we don't want to support it in the future.

Reply:
done